Testing/VelaptorTests/OpenGL/GLWindowTests.cs

```diff
@@ -798,27 +805,30 @@ public void TypeOfBorder_WhenGettingInvalidValue_ThrowsException()
         var act = () => _ = sut.TypeOfBorder;
 
         // Assert
-        act.Should().Throw<EnumOutOfRangeException<Silk.NET.Windowing.WindowBorder>>()
+        act.Should()
+            .Throw<InvalidEnumArgumentException>()
             .WithMessage(expected);
     }
 
     [Fact]
     public void TypeOfBorder_WhenSettingInvalidValue_ThrowsException()
     {
         // Arrange
-        var expected = $"The value of the enum '{nameof(Velaptor.WindowBorder)}' used in the class '{nameof(GLWindow)}' and";
-        expected += " method 'SetupOtherPropCaches' is invalid and out of range.";
+        var invalidValue = 1234;
```

Comment:
Please refactor to a constant

```suggestion
        const invalidValue = 1234;
```